incremental-font-transfer/src/uri_templates.rs

```diff
@@ -8,9 +8,13 @@
 //!
 //! By implementing our own we avoid pulling in a much larger general purpose template expansion library
 //! and improve performance versus a more general implementation.
+use data_encoding::BASE64URL;
+use data_encoding_macro::new_encoding;
```

Comment:
I wonder why your imports never end up getting combined? Maybe this is some setting in rust-analyzer..

Reply:
oops 😅